magic/__init__.py

```diff
@@ -15,6 +15,7 @@
 >>>
 
 """
+from __future__ import annotations
```

Comment:
setup.py currently allows python 2.7 and 3.5+, where this will fail I think

Reply:
Python 2 died 1,603 days ago on 1/1/2020.  Continuing to support it these daze only slows development.

It is probably high time to say that `v0.4.27` is the last [release](https://github.com/ahupp/python-magic/tags) to support Python 2.7.

Given that the GitHub Actions only test [supported versions of Python](https://devguide.python.org/versions) how do we prove that legacy Python still works?